index.html

```diff
@@ -29,19 +29,21 @@
                             17 10H1Z" fill="#6070FF" />
                 </svg>
                 <ul class="menu show-if-large hidden-if-small">
-                    <li class="p-for-textbox"><span class="text">Portfolio</span></li>
-                    <li class="p-for-textbox"><span class="text">About</span></li>
-                    <li class="p-for-textbox"><span class="text">Contact</span></li>
+                    <li class="p-for-textbox"><a href="#"><span class="text">Portfolio</span></a></li>
+                    <li class="p-for-textbox"><a href="#"><span class="text">About</span></a></li>
+                    <li class="p-for-textbox"><a href="#"><span class="text">Contact</span></a></li>
                 </ul>
             </nav>
         </div>
     </header>
     <section class="header-background content-position">
+        <img src="resources/Header bg.svg" alt="not_working" class="background hidden-if-small"/>
         <div class="animation-loader position-loader-1"></div>
         <div class="animate-box position-box-1"></div>
-        <div class="primary-text-container">
+        <div class="primary-text-container m-p-19">
             <h1 class="primary-text1 margin-0">
-                I&#39m <span class="animate-header">Mariane</span> <br />Glad to see you!
+                I&#39m <span class="animate-header hidden-if-large">Mariane</span>
+                <span class="animate-header hidden-if-small">Alexandra</span> <br />Glad to see you!
```

Comment:
[OPTIONAL] Your project is looking awesome! Give it a personal look by putting your name instead of the placeholder one. You can also customize all the information to be about yourself (and display your own projects!). 😉

Reply:
Fixed.